courses/machine_learning/deepdive2/introduction_to_tensorflow/labs/4_keras_functional_api.ipynb

```diff
@@ -159,6 +159,16 @@
     "For the Wide columns, we will create feature columns of crossed features. To do this, we'll create a collection of Tensorflow feature columns to pass to the `tf.feature_column.crossed_column` constructor. The Deep columns will consist of numeric columns and the embedding columns we want to create. "
```

Comment:
<p>Depending what you do, just flagging the TODO numbers here in case you change :)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/704/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
<p>done</p>